extension/js/common/ui/key-import-ui.ts

```diff
@@ -73,15 +74,36 @@ export class KeyImportUi {
       } else if ((this as HTMLInputElement).value === 'backup') {
         window.location.href = Url.create('/chrome/settings/setup.htm', { acctEmail, parentTabId, action: 'add_key' });
       }
+      if (view !== undefined && view.submitKeyForAddrs.length > 1) {
+        $('.container_for_import_key_email_alias').css('visibility', 'visible');
+      }
     });
     $('.line.unprotected_key_create_pass_phrase .action_use_random_pass_phrase').click(Ui.event.handle(() => {
       $('.source_paste_container .input_passphrase').val(PgpPwd.random()).keyup();
       $('.input_passphrase').attr('type', 'text');
       $('#e_rememberPassphrase').prop('checked', true);
     }));
+    $('.input_private_key').on('keyup paste change', Ui.event.handle(async target => {
+      $('.action_add_private_key').addClass('btn_disabled').attr('disabled');
+      $('.input_email_alias').prop('checked', false);
+      const { keys: [prv] } = await opgp.key.readArmored(String($(target).val()));
+      if (prv !== undefined) {
+        $('.action_add_private_key').removeClass('btn_disabled').removeAttr('disabled');
+        if (view !== undefined && view.submitKeyForAddrs.length > 1) {
+          const users = prv.users;
+          for (const user of users) {
+            const userId = user.userId;
+            for (const inputCheckboxElement of $('.input_email_alias')) {
+              if (String($(inputCheckboxElement).data('email')) === userId!.email) {
+                $(inputCheckboxElement).prop('checked', true);
```

Comment:
so similar to the handler we have in `setup-render.ts` we should modify `submitKeyForAddrs`
```
    $('.input_email_alias').click((event) => {
      const email = String($(event.target).data('email'));
      if ($(event.target).prop('checked')) {
        if (!this.view.submitKeyForAddrs.includes(email)) {
          this.view.submitKeyForAddrs.push(email);
        }
      } else {
        this.view.submitKeyForAddrs.splice(this.view.submitKeyForAddrs.indexOf(email), 1);
      }
    });
    ```
    You can add helper functions addAliasForSubmission and removeAliasFromSubmission and use them from all the relevant places, but please make `setup-render.ts` referenbce this script, not the other way around.